db/db_with_timestamp_basic_test.cc

```diff
@@ -704,6 +704,60 @@ TEST_F(DBBasicTestWithTimestamp, SimpleIterate) {
   Close();
 }
 
+TEST_F(DBBasicTestWithTimestamp, TrimHistoryTest) {
+  Options options = CurrentOptions();
+  options.env = env_;
+  options.create_if_missing = true;
+  const size_t kTimestampSize = Timestamp(0, 0).size();
+  TestComparator test_cmp(kTimestampSize);
+  options.comparator = &test_cmp;
+  DestroyAndReopen(options);
+  // Construct data of different versions with different ts
+  WriteOptions wopts;
+  std::string ts_str = Timestamp(2, 0);
+  Slice ts = ts_str;
+  ASSERT_OK(db_->Put(wopts, "k1", ts, "v1"));
```

Comment:
Due to implicit conversion from `std::string` to `Slice`, we can just do
```
ASSERT_OK(db_->Put(WriteOptions(), "k1", Timestamp(2, 0), "v1"));
```
Also, just use `WriteOptions()` if all default. Performance is not a concern here, and this way we do not have to track the value of `wopts`.

Reply:
This applies to the rest of this test.

Reply:
fixed